src/main/java/io/github/opencubicchunks/cubicchunks/core/server/VanillaNetworkHandler.java

```diff
@@ -210,50 +252,74 @@ public void sendBlockChanges(TShortList dirtyBlocks, Cube cube, EntityPlayerMP p
                             localToBlock(cube.getZ(), z)));
         }
         ((ISPacketMultiBlockChange) packet).setChangedBlocks(updates);
-        ((ISPacketMultiBlockChange) packet).setChunkPos(cube.getCoords().chunkPos());
+        ((ISPacketMultiBlockChange) packet).setChunkPos(new ChunkPos(posX, posZ));
 
         player.connection.sendPacket(packet);
     }
 
-    public void updatePlayerPosition(PlayerCubeMap cubeMap, EntityPlayerMP player, int managedPosY) {
+    public void updatePlayerPosition(PlayerCubeMap cubeMap, EntityPlayerMP player, CubePos managedPos, int teleportId) {
```

Comment:
Aside the force recheck one (which I don't think is necessary), there is no longer any reason for the teleportId to exist here